fe/src/pages/index.vue

```diff
@@ -1,6 +1,41 @@
 <template>
   <div>
-    WELCOME TO VUE!
-    <BaseDropdown />
+    <nav class="h-[64px] border-b grid place-content-center">Navbar</nav>
```

Comment:
navclass/navbar is used across pages. Please transfer to `layout/default.vue` instead.